test/NosCore.Tests/BazaarTests/BazaarTests.cs

```diff
@@ -380,24 +391,24 @@ public void ModifyBazaar()
                     IsPackage = false,
                     ItemInstanceId = _guid,
                     Price = 80
-                });
+                }).ConfigureAwait(false);
             var patch = new JsonPatchDocument<BazaarLink>();
             patch.Replace(link => link.BazaarItem!.Price, 50);
-            Assert.IsNotNull(_bazaarController.ModifyBazaar(0, patch));
+            Assert.IsNotNull(await _bazaarController.ModifyBazaarAsync(0, patch).ConfigureAwait(false));
             Assert.AreEqual(50, _bazaarItemsHolder?.BazaarItems[0].BazaarItem?.Price);
         }
 
         [TestMethod]
-        public void ModifyBazaarAlreadySold()
+        public async Task ModifyBazaarAlreadySoldAsync()
         {
             _mockItemDao!
-                .Setup(s => s.FirstOrDefault(It.IsAny<Expression<Func<IItemInstanceDto, bool>>>()))
-                .Returns(new ItemInstanceDto
+                .Setup(s => s.FirstOrDefaultAsync(It.IsAny<Expression<Func<IItemInstanceDto?, bool>>>()))
+                .ReturnsAsync(new ItemInstanceDto
                 {
                     Id = _guid,
                     Amount = 99
                 });
-            var add = _bazaarController!.AddBazaar(
+            var add = await _bazaarController!.AddBazaarAsync(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'add'.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5326532)